dist/stat_data.py

```diff
@@ -315,7 +315,7 @@ def __init__(self, name, desc, flags=''):
     YieldStat('page_sleep', 'page acquire time sleeping (usecs)'),
 ]
 
-connection_stats = sorted(connection_stats, key=attrgetter('name'))
+connection_stats = sorted(connection_stats, key=attrgetter('desc'))
```

Comment:
These changes resulted in most of the line movement -- I needed the statistics logically grouped rather than being returned in order of the internal names we give them.  I think it makes more sense anyway: our names aren't chosen consistently to group stats together but the type prefixes in the description are.